Sources/Database/Database.swift

```diff
@@ -1070,6 +1070,45 @@ private struct _Client {
         ON "episode_progresses" ("episode_sequence", "user_id")
         """
       )))
+      .flatMap(const(execute(
+        """
+      CREATE SEQUENCE IF NOT EXISTS users_referral_code_sequence
```

Comment:
This is to create a small auto-incrementing sequence in the rare case that two people create their accounts at the exact same second...